doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1220 @@
+
+* Updated by enabling Link Time Optimization (LTO) by default for the ``release`` configuration.
+* Removed support for the nRF54H20 devices.
+
+nRF5340 Audio
+-------------
+
+* Updated:
+
+  * The documentation for :ref:`nrf53_audio_app_building` with cross-links and additional information.
+  * The :file:`buildprog.py` script is an app-specific script for building and programming multiple kits and cores with various audio application configurations.
+    The script will be deprecated in a future release.
+    The audio applications will gradually shift only to using standard tools for building and programming development kits.
+  * The :ref:`nRF5340 Audio application\'s <nrf53_audio_app>` :ref:`script for building and programming <nrf53_audio_app_building_script>` now builds into a directory for each transport, device type, core, and version combination.
+
+* Fixed the static random address for broadcast source and unicast server.
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * System Power Management for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target on the application and radio cores.
+  * Application configurations for the nRF54L05 and nRF54L10 SoCs (emulated on the nRF54L15 DK).
+    The configurations are supported through ``nrf54l15dk/nrf54l10/cpuapp`` and ``nrf54l15dk/nrf54l05/cpuapp`` board targets.
+    For details, see the :ref:`nrf_desktop_board_configuration`.
+  * The ``dongle_small`` configuration for the nRF52833 DK.
+    The configuration enables logs and mimics the dongle configuration used for small SoCs.
+  * Requirement for zero latency in Zephyr's :ref:`zephyr:pm-system` while USB is active (:ref:`CONFIG_DESKTOP_USB_PM_REQ_NO_PM_LATENCY <config_desktop_app_options>` Kconfig option of the :ref:`nrf_desktop_usb_state_pm`).
+    The feature is enabled by default if Zephyr power management (:kconfig:option:`CONFIG_PM`) is enabled.
+    It prevents entering power states that introduce wakeup latency and ensures high performance.
+  * Static Partition Manager memory maps for single-image configurations (without bootloader and separate radio/network core image).
+    In the |NCS|, the Partition Manager is enabled by default for single-image sysbuild builds.
+    The static memory map ensures control over settings partition placement and size.
+    The introduced static memory maps might not be consistent with the ``storage_partition`` defined by the board-level DTS configuration.
+  * Support for GATT long (reliable) writes (:kconfig:option:`CONFIG_BT_ATT_PREPARE_COUNT`) to Fast Pair and Works With ChromeBook (WWCB) configurations.
+    This allows performing :ref:`fwupd <nrf_desktop_fwupd>` DFU image upload over Bluetooth LE with GATT clients that do not perform MTU exchange (for example, ChromeOS using the Floss Bluetooth stack).
+  * The ``dongle`` and ``release_dongle`` application configurations for the nRF54H20 Development Kit (``nrf54h20dk/nrf54h20/cpuapp``).
+    The configurations act as a HID dongle.
+
+* Updated:
```

Comment:
Please add under nRF Desktop updated (assuming https://github.com/nrfconnect/sdk-nrf/pull/21466 goes in):
```
  * Disabled the UDC DWC2 DMA support (:kconfig:option:`CONFIG_UDC_DWC2_DMA`) on nRF54H20 DK.
    The DMA support is experimental, disabling the feature improves USB HID stability.
    Since nRF Desktop uses only small HID reports (report size is smaller than 64 bytes), the DMA doesn't improve performance.
```

Reply:
Added.